packages/graphile-build-pg/src/plugins/PgIntrospectionPlugin.js

```diff
@@ -199,6 +199,23 @@ export type PgEntity =
   | PgExtension
   | PgIndex;
 
+export type PgIntrospectionResultsByKind = {
```

Comment:
Please use this type; e.g. as the return value for the `introspect` function.

Reply:
how do I do that? should I annotate that on the js file? or export a function overload in d.ts file?

Reply:
Yes; this is using flowtype so you can do it similarly to TypeScript:


```js
  async function introspect(): Promise<PgIntrospectionResultsByKind> {
```

I just figure without that the type isn't actually _used_ anywhere, so it's value is less (and it's likely to come out of sync over time).

Reply:
it is pretty hard tho, because the `introspect` is not exported. The `d.ts` file is really used by users to know the module's exported member's type.  So I added a jsDoc annotation to the `introspect` function, now it will be checked by typescript. I also added export of the function type, and the export default function